lib/quantizeAttributes.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var addExtensionsUsed = require('./addExtensionsUsed');
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var packBuffers = require('./packBuffers');
+
+module.exports = quantizeAttributes;
+
+/**
+ * Quantizes attributes in the provided glTF using the WEB3D_quantized_attributes extension.
+ * If options is undefined, all quantizable attributes will be quantized with the default options.
+ * If options.attributes is undefined, all quantizable attributes will be quantized with the provided options.
+ * If options.precision is undefined, the decodeMatrix be written in its full form.
+ *
+ * @param {Object} [gltf] A javascript object holding a glTF hierarchy.
+ * @param {Object} [options=undefined] Defines more specific quantization behavior.
+ * @param {Object} [options.semantics=undefined] Defines which semantics should be quantized.
+ * @param {Number} [options.precision=undefined] Restricts the number of decimal places in the decodeMatrix.
+ *
+ * @returns glTF with quantized attributes
+ *
+ * @example
+ * var gltf = {
+ *     accessors: {
+ *         accessor_0: {...}
+ *         accessor_1: {...}
+ *     },
+ *     meshes: {
```

Comment:
I'll remove the example gltf hierarchy at the top though. It isn't really necessary anymore since the examples aren't targeting specific id's anymore